### ISSUE TITLE
Zoobank - scientificNameAuthorship empty values filled with the string: <Unspecified Agent>

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/xcol#31]
dwc:scientificNameAuthorship has empty values as: "< Unspecified Agent >", 46902 records (11%) are affected by it, 43299 genus, 173 species, 1 subspecies.

See: https://www.checklistbank.org/dataset/2037/taxon/bbb21db2-66ac-47ea-836f-805854c026d8

**Comment by mdoering** [comment]
@deepreef any chance to remove the unspecified agent from the export?
There are also other bad names that use `[none]` or  odd accordingTo values like `sensu 1999.… 1999`:

- tobedeleted
- tobedeleted , 1999
- Scelotes tobedeleted , 1999
- tobedeleted1 , 1999
- Microdus (Therophilus) (delete) , 1999
- Euchromia (grp. tobedeleted) , 1999
- tobedeleted4 Wiegmann, 1828
- delete , 1999 [none] 1999 sensu 1999.… 1999
- delete exaratus [DELETE] [none], 1999 sec. [none]
- [Genus?] delete [[none], 1999] sec. [none]
- delete exaratus , 1999 [none] 1999 sensu 1999.… 1999
- delete exaratus [DELETE] [none], 1999 sec. [none]

**Comment by mdoering** [comment]
see also https://github.com/CatalogueOfLife/xcol/issues/34

**Comment by deepreef** [comment]
Many thanks for pointing these out.  There is a LOT of clean-up we need to do, and that will be included in the grant proposal later this year.  But if there are particularly problematic records, like the ones above, I can probably set aside some time next weekend to clean them up.

I'm not sure exactly what you mean by removing the Unspecified Agent from the export -- do you mean in terms of that agent as an author of exported publications?

**Comment by camiplata** [comment]
yes @deepreef, we mean to remove Unspecified Agent as an author to avoid combinations such as:

<img width="536" alt="Captura de pantalla 2023-07-31 a la(s) 8 33 08 a m" src="https://github.com/CatalogueOfLife/xcol/assets/5700527/3a4470b8-801a-4443-ad0e-6d853d4ff1ec">

**Comment by deepreef** [comment]
OK, I can change the way the author is presented when not known.  This is a tag we use to indicate that we have not tracked down the original publication.  But if it would be better to present a Null value in the output, I can certainly do that.

**Comment by camiplata** [comment]
Dear @deepreef, we would like to know if it is possible to remove the placeholder <Unspecified Agent> in the Authorship in the next version of Zoobank dataset. If needed we can provide assistance with these changes.
